### PR TITLE
Update prerequisites.md

### PR DESCRIPTION
**Pull request by msbemba**
Added mailbox requirements

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/4186

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/mtp/prerequisites.md [hunk 1]
```diff
@@ -71,6 +71,17 @@ Currently, Microsoft 365 Defender is *not* available to:
 - US Department of Defense
 - All US government institutions with commercial licenses
 
+## Mailbox Requirements
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/prerequisites.md]:
```suggestion
## Mailbox requirements
```

File: microsoft-365/security/mtp/prerequisites.md [hunk 2]
```diff
@@ -71,6 +71,17 @@ Currently, Microsoft 365 Defender is *not* available to:
...
+
+The following are the primary ways you can use Microsoft 365 Defender for message protection:
+
+- In a Microsoft 365 Defender filtering-only scenario, Defender provides cloud-based email protection for your on-premises Exchange Server environment or any other on-premises SMTP email solution.
+
+- Microsoft 365 Defender can be enabled to protect Exchange Online cloud-hosted mailboxes. To learn more about Exchange Online, see the Exchange Online service description.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/mtp/prerequisites.md]:
```suggestion
- Microsoft 365 Defender can be enabled to protect Exchange Online cloud-hosted mailboxes. To learn more about Exchange Online, see the [Exchange Online service description](https://docs.microsoft.com/office365/servicedescriptions/exchange-online-service-description/exchange-online-service-description).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@mjcaparas Gentel reminder, Please merge this PR. Thanks.

**Comment by mjcaparas** [comment]
pls resolve conflicts. then i can merge.

**Comment by msbemba** [comment]
@yogkumgit  @JohanFreelancer9  for some reason I don't see an option to resolve conflicts and it is grayed out (saying Only those with the write access can merge pull requests). Let me know what to do.

**Comment by yogkumgit** [comment]
@Kateyanne Please provide the internal help. Thanks.

**Comment by msbemba** [comment]
@yogkumgit The changes are made on the document already. So this PR is no longer required.

**Pull request closed without merging**